doc/contributing/building_ruby.md

```diff
@@ -0,0 +1,77 @@
+# Building Ruby
+
+## Quick start guide
+
+1. Install the prerequisite dependencies for building the CRuby interpreter:
+
+    * C compiler
+    * autoconf - 2.67 or later
+    * bison - 2.0 or later
+    * gperf - 3.0.3 or later
+    * ruby - 2.7 or later
+
+2. Install optional, recommended dependencies:
+
+    * OpenSSL/LibreSSL
+    * readline/editline(libedit
+    * zlib
+    * libffi
+    * libyaml
+    * libexecinfo (FreeBSD)
+    * NDBM/QDBM
```

Comment:
The dbm and gdbm extensions were removed, so the last two bullets can be removed